apps/openssl.cnf

```diff
@@ -16,6 +16,10 @@ HOME			= .
  # Use this in order to automatically load providers.
 openssl_conf = openssl_init
 
+# Comment out this if you deliberately want to ignore
+# the configurations error
```

Comment:
s/Comment out this/Comment this out/
s/ignore the configurations error/ignore configuration errors/

Reply:
Done in both files.